wire-ios-data-model/Source/MLS/CoreCryptoKeyProvider.swift

```diff
@@ -26,7 +26,13 @@ public class CoreCryptoKeyProvider {
     }
 
     public func coreCryptoKey() throws -> Data {
+        removeLegacyKeyIfNeeded()
+        return try fetchOrCreateCoreCryptoKey()
+    }
+
+    private func fetchOrCreateCoreCryptoKey() throws -> Data {
```

Comment:
Perhaps this should be:

```swift
public func coreCryptoKey(createIfNeeded: Bool = false) throws -> Data
```

so we can make sure we only try to create it from the main app and not the extensions.

Reply:
addressed in [bbac05a](https://github.com/wireapp/wire-ios-mono/pull/100/commits/bbac05a0e4022ba7d4bc7bdc80020925b1bb842c)